migrations.lua

```diff
@@ -6,6 +6,8 @@ return {
       { "id", types.serial {unique = true, primary_key = true }},
       { "name", types.text {unique = true}},
       { "email", types.text {unique = true}},
+      { "password", types.varchar },
```

Comment:
Why the varchar vs text?

Reply:
Because in my brief research I saw it recommended to use this data type for storing hashed passwords. Nothing more than that really.